papyri/common_ast.py

```diff
@@ -81,6 +81,16 @@ def __hash__(self):
         )
 
 
+class UnserializableNode(Node):
+    _dont_serialise = True
+
+    def cbor(self, encoder):
+        assert False, self
```

Comment:
Yeah, I think we can just add the part that fail on a table on hte ignore list. I don't think tree-sitter-rst plan to parse tables, and I'm wondering if having a `.. table::` directive could be a good workaround, we can add it to sphinx as well, so that projects work on both papyri and sphinx at the same time.